templates/fonts/swift5.stencil

```diff
@@ -70,8 +70,8 @@
   }
 
   fileprivate var url: URL? {
-    let bundle = BundleToken.bundle
-    return bundle.url(forResource: path, withExtension: nil)
+    // swiftlint:disable:next implicit_return
+    return {{param.bundle|default:"BundleToken.bundle"}}.url(forResource: path, withExtension: nil)
```

Comment:
I'm guessing you didn't use implicit_return in order for the template to be compatible with Swift 5.0 (since they were added in 5.1); was wondering if there were a risk of other places where we might have used implicit returns making our generated code not compatible with 5.0?

I'm not entirely sure to remember how our `compilation-configuration.yml` files work but if we have `swift_versions: [5]` in it, I think it just uses a `SWIFT_VERSION` of `5` in Xcode… which only uses the latest Swift 5.x sub-version shipped with that Xcode SDK, so currently 5.2, and doesn't test with Swift 5.0? If that's the case that we could easily have missed other places and 5.1+-only features?

Reply:
Well, yeah, exactly. Thing is, we can't test it, unless we add something on Circle to run it on older Xcode versions, but then... Probably bunch of other issues.

Luckily, SwiftLint helps us out here. It wants us to remove the `return` because we can in Swift 5.1+, but **we** can't, so it we disable the rule here. If we at some point **fix** the code, then SwiftLint will warn us about a useless disable.

Reply:
Ah, good point. Perfect then, that's reassuring 🙂